packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationsEditor.test.tsx

```diff
@@ -212,41 +213,38 @@ describe('SchemaDrivenAnnotationEditor tests', () => {
       // Fill in annotations that match the schema in this test suite
       response.showStringArray = true
       delete response.stringArray
-      return res(ctx.status(200), ctx.json(response))
+      return HttpResponse.json(response, { status: 200 })
     },
   )
 
-  const noSchemaHandler = rest.get(
+  const noSchemaHandler = http.get(
     `${getEndpoint(
       BackendDestinationEnum.REPO_ENDPOINT,
     )}${ENTITY_SCHEMA_BINDING(':entityId')}`,
-    async (req, res, ctx) => {
-      return res(ctx.status(404), ctx.json({}))
+    () => {
+      return HttpResponse.json({}, { status: 404 })
     },
   )
 
   const schemaHandlers = [configureSchemaBindingHandler(mockSchemaBinding)]
 
-  const successfulUpdateHandler = rest.put(
+  const successfulUpdateHandler = http.put(
     `${getEndpoint(BackendDestinationEnum.REPO_ENDPOINT)}${ENTITY_JSON(
       ':entityId',
     )}`,
-    async (req, res, ctx) => {
-      updatedJsonCaptor(req.body)
```

Comment:
These attempts to capture the request object in the HTTP service broke after upgrading (and were often tricky to set up in tests anyways). Instead of trying to figure out how to use them, I just decided to capture the object passed to the `SynapseClient` method.